src/pal/src/arch/arm/exceptionhelper.S

```diff
@@ -22,12 +22,12 @@ LEAF_ENTRY ThrowExceptionFromContextInternal, _TEXT
     ldr	r10,	[r0, #(CONTEXT_R10)]
     ldr	r11,	[r0, #(CONTEXT_R11)]
     ldr	sp,	[r0, #(CONTEXT_Sp)]
-    ldr	lr,	[r0, #(CONTEXT_Lr)]
 
     ldr	r2,	[r0, #(CONTEXT_Pc)]
 
     // Store return address to the stack
-    push    {r2}
+    // Added r7 as a dummy to keep the stack aligned by 8 bytes.
+    push    {r2, r7}
```

Comment:
Traced until PC hits `restore_core_regs` and goes back to the code that generated the exception/signal and never found a code that "pop"s this SP pointer. So I think it's harmless to add this dummy (or maybe even remove this push statement?).